internal/conditions/cel.go

```diff
@@ -17,17 +17,18 @@ import (
 )
 
 const (
-	CELRequestIdent    = "request"
-	CELResourceAbbrev  = "R"
-	CELResourceField   = "resource"
-	CELPrincipalAbbrev = "P"
-	CELPrincipalField  = "principal"
-	CELRuntimeIdent    = "runtime"
-	CELVariablesIdent  = "variables"
-	CELVariablesAbbrev = "V"
-	CELGlobalsIdent    = "globals"
-	CELGlobalsAbbrev   = "G"
-	CELAttrField       = "attr"
+	CELRequestIdent      = "request"
+	CELResourceAbbrev    = "R"
+	CELResourceKindField = "kind"
```

Comment:
I think the `id` field is accessible in conditions too. Is that handled as well? https://github.com/cerbos/cerbos/blob/002b035da064ea5b6a78d8f86552a9e71683fb62/api/public/cerbos/engine/v1/engine.proto#L357-L360

Reply:
The planner uses another `Resource` type without ID: https://github.com/cerbos/cerbos/blob/002b035da064ea5b6a78d8f86552a9e71683fb62/api/public/cerbos/engine/v1/engine.proto#L20-L60